src/pymedphys/docker/nginx/nginx.conf

```diff
@@ -0,0 +1,87 @@
+events {
+    worker_connections 1024;
+}
+
+stream {
+    upstream backend {
+        server rccc-physicssvr:104;
```

Comment:
There are some RCCC specific items in here. In the long term I desire to pull these out into a configuration file, which can then be edited via a GUI.